### DISCUSSION TITLE
Difficult to Read Syntax Highlighting for Golang with Symbols Pane

### DISCUSSION THREAD

**Discussion by dbc60** [community/community#125425] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

When I use the symbol pane to find a particular Go symbol, it's highlighted in such a way that the symbol is nearly unreadable. Is there any way to fix/adjust it? Thanks.

Here's what it looks like in Chrome on Windows:

![github-symbols-highlighting](https://github.com/community/community/assets/2331247/d283ef67-c0b1-4a48-bf6e-fbbfad84dfab)

**Comment by AdamShwert** [comment, marked as answer]
Which coloring theme/contrast mode are you using? We'll see if we can get the colors adjusted!

**Reply by dbc60** [reply]
My settings say I'm using the "Night theme". I just switched to the Light default theme. It's easier to read now, but the contrast could be better here, too. Thanks for looking into it.

**Reply by dbc60** [reply]
FWIW, it's much easier to read with the Dark Protanopia & Deuteranopia theme.
